### DISCUSSION TITLE
Secrets management?

### DISCUSSION THREAD

**Discussion by automato2000** [microsoft/react-native-windows#10965] [Q&A] [answered]
Hi there,
I'm building a RN Windows app and have run into a stopping point where I need to figure out a secure way to store application secrets (passwords, signing/private keys, API tokens, etc.) so that they're only accessible to the Windows user that's using my app during runtime only. Is there a recommended way to do this? 

Thanks!

**Comment by chrisglein** [comment, marked as answer]
You'll want to look around for how to do this in UWP, independent of RN. Some hits I found:
- https://stackoverflow.com/questions/36530622/what-is-the-recommened-way-to-store-api-keys-and-secrets-in-a-uwp-app
- https://learn.microsoft.com/en-us/windows/uwp/security/intro-to-secure-windows-app-development
